translations/en.json

```diff
@@ -1221,5 +1221,20 @@
 			"n-year-ago": "{{ timeDelta }}y ago"
 		},
 		"min": "min"
+	},
+	"mobile-menu": {
+		"title-menu": "Menu",
```

Comment:
More of a design nitpick, but not sure we even need this menu to say "Menu". I feel like users understand the hamburger menu enough. @asherism

Reply:
<img width="498" alt="image" src="https://user-images.githubusercontent.com/10401554/170377945-4015ea41-300d-4fb5-b1bc-73a97a137fb4.png">